builtin/game/falling.lua

```diff
@@ -30,9 +30,19 @@ core.register_entity(":__builtin:falling_node", {
 	end,
 
 	get_staticdata = function(self)
+		local meta = self.meta
+		if meta and meta.inventory then
+			for k1, v1 in pairs(meta.inventory) do
+				for k2, v2 in pairs(v1) do
+					if type(v2) == "userdata" then
+						v1[k2] = v2:to_string()
+					end
+				end
+			end
+		end
 		local ds = {
 			node = self.node,
-			meta = self.meta,
+			meta = meta,
```

Comment:
`self.meta` and `meta` is the same table, it was not copied.
(Maybe it would be better to do all this in `set_node`, idk.)

Reply:
I just realized that it would be better if minetest.serialize can parse inventory meta data into a string.